docs/configuration/config-file-reference.md

```diff
@@ -1499,6 +1499,55 @@ filesystem:
   # CLI flag: -local.chunk-directory
   [directory: <string> | default = ""]
 
+elastic:
+  # Address of ElasticSearch.
+  # CLI flag: -elastic.address
+  [address: <string> | default = "http://127.0.0.1:9200"]
+
+  # Template name in ElasticSearch.
+  # CLI flag: -elastic.template_name
```

Comment:
Please make sure CLI flags are `-` separated, while config file are `_` separated. In this case, the config file option name `template_name` is fine, while the CLI flag should be `-elastic.template-name`.